src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
should be same as color supplier